test/internalTest.js

```diff
@@ -30,6 +30,7 @@ mineflayer.supportedVersions.forEach((supportedVersion, i) => {
           username: 'player',
           version: supportedVersion,
           port: 25567
+          // logErrors: false
```

Comment:
Why ?

Reply:
I was debugging why my test does not work, going to remove it quickly